### ISSUE TITLE
Info: Angular 21 / Zoneless Compatible Version available on fork

### ISSUE THREAD

**Issue by PascalHonegger** [mariuszfoltak/angular2-datatable#171]
For anyone that's interested, my fork over at [https://github.com/PascalHonegger/ng-datatable](https://github.com/PascalHonegger/ng-datatable) **supports Angular 11 and ivy**. The build pipeline now uses the Angular CLI and should allow for upgrades to future Angular versions as well.
If you still want to use (or maybe even improve) this project, you're welcome to use the fork or submit a pull request 😉.

*This isn't an issue, I still believe it's a nice thing to help people looking into Angular 11 / ivy problems*

**Comment by danielehrhardt** [comment]
Could you make a pull request?

**Comment by PascalHonegger** [comment]
This project seems abandoned and other pull requests have been ignored for years, which is why I didn't create a pull request

**Comment by danielehrhardt** [comment]
> This project seems abandoned and other pull requests have been ignored for years, which is why I didn't create a pull request

Make sense 👍

**Comment by SiirGustav** [comment]
Yes, thank you very much! I was pulling my hair out because I had to update an Angular 8 project, and all the tables were using this module. I was looking at the forks but those were outdated as well, I'm grateful I came to look at the issues, your fork works like a charm!
